README.md

```diff
@@ -321,11 +321,39 @@ To install things on the internal MiR PC:
 * connect a monitor and keyboard to the ports that are exposed on one corner of the MiR
 * boot into a live USB linux system
 * `chroot` into the MiR PC
-* download `chrony_2.1.1-1ubuntu0.1_amd64.deb`,
-  `libtomcrypt0_1.17-7ubuntu0.1_amd64.deb`, `libtommath0_0.42.0-1.2_amd64.deb`
-  and `timelimit_1.8-1_amd64.deb` from a PC that has internet and install them
-  in the `chroot` environment onto the MiR PC using `dpkg -i`
+    1. mount MiR partition and bind /dev and /run:
+       ```
+       sudo mkdir -p /media/mir
+       sudo mount /media/mir/ /dev/sda3
+       sudo mount --bind /dev/ /media/mir/@/dev
+       sudo mount --bind /run/ /media/mir/@/run
+       ```
+    2. `chroot` into the MiR PC:
+       ```sudo chroot /media/mir/@/ ```
+* crate user:
+   ```
+   adduser newuser
+   usermod -aG sudo newuser 
+   ```
+* reboot and login into MiR PC
```

Comment:
Usually, the MiR PC doesn't have a connection to the internet (at least on our robot), so I guess we should install chrony *before* rebooting (assuming the live USB system has an internet connection).

Reply:
All of our MiRs have internet connection, therefore I did it this way. I know this way works 100%. By installing via the chroot enviroment, dns resolving wont work. So you probably have to copy /etc/resolv.conf to use apt. But because I'm not sure if I miss something and didn't try it on the HW, I don't want to change this in the manual.